packages/graphback-core/src/scalars/index.ts

```diff
@@ -1,2 +1,70 @@
-export * from 'graphql-type-json';
-export * from './scalars'
+import { GraphQLNamedType, GraphQLScalarType } from 'graphql';
+import { TimeResolver, TimestampResolver, DateResolver, DateTimeResolver, ObjectIDResolver, JSONResolver, JSONObjectResolver } from 'graphql-scalars';
+
+export const GraphbackTime = new GraphQLScalarType({
```

Comment:
Would it be possible to have those in schema plugin. Feels like specific to this plugin to me

Reply:
No strong opinion on this one. I put them on the core maybe because some other plugins would want to benefit from those and act on them. Also because the file was already there. @craicoverflow what do you think?

Reply:
Changed mind. +1 on keeping them